inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/config.cpp

```diff
@@ -42,18 +42,7 @@ namespace {
     };
 
     const std::vector<std::map<std::string, std::string>> AutoConfigs = {
-            {{InferenceEngine::KEY_AUTO_DEVICE_LIST , CommonTestUtils::DEVICE_CPU},
-                    {InferenceEngine::PluginConfigParams::KEY_CPU_THROUGHPUT_STREAMS, InferenceEngine::PluginConfigParams::CPU_THROUGHPUT_AUTO}},
```

Comment:
maybe it was better to keep the test cases, but implement the variant of the InferConfigTests that sets the options via the ie.Setconfig directly to the plugin specified in the KEY_AUTO_DEVICE_LIST, as opposite to passing them to the LoadNetwork

Reply:
Thanks Maxim, I didn't delete these tests but only remove CPU related config for AUTO plugin. and they have no difference by ie.SetConfig(config) or LoadNetwork(config) since both will ignore thoese config for CPU.

Reply:
I was referring to setting config with ie.SetConfig(config, CPU) directly to the device, and then testing that AUTO works

Reply:
The tests are in plugin/shared folder, so it may be used by other plugin. I think we could not add ie.SetConfig(config, CPU) to those common test code.